demo/src/app/components/popover/popover.component.ts

```diff
@@ -18,6 +18,10 @@ import {DEMO_SNIPPETS} from './demos';
         demoTitle="Custom and manual triggers" [snippets]="snippets" component="popover" demo="triggers">
         <ngbd-popover-triggers></ngbd-popover-triggers>
       </ngbd-example-box>
+      <ngbd-example-box
+        demoTitle="AutoClose" [snippets]="snippets" component="popover" demo="autoclose">
```

Comment:
`demoTitle="Closing with mouse and keyboard"` or something similar?

Reply:
Agreed